app/models/metric/ci_mixin/state_finders.rb

```diff
@@ -1,16 +1,18 @@
 module Metric::CiMixin::StateFinders
+  # @param ts [Time|String] prefer Time
   def vim_performance_state_for_ts(ts)
+    ts = Time.parse(ts).utc if ts.kind_of?(String)
```

Comment:
Are any callers still passing string objects?  It looks like you are now passing Time objects in this PR.

Reply:
I tried to make the switch, but timestamps are a very big mix of strings vs timestamps. Some only accept strings and others only accept time objects.